marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserModulesCommand.java

```diff
@@ -180,6 +177,12 @@ public void execute(CommandContext context) {
         // load any user files under plugins/* int the modules database.
         // this will ignore REST folders under entities
         DefaultModulesLoader modulesLoader = getStagingModulesLoader(config);
+        // Load modules from standard ml-gradle location after 'PropertiesModuleManager' is initialized. This will ensure
+        // that in case of 'forceLoad' the ml-javaclient-utils timestamp file is deleted first and we the old timestamp file
```

Comment:
Need to fix the grammar in the comment "and we the old timestamp file is not used when loading" - actually, I think you can end the comment right after "is deleted first".